jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>factors in the sense of categoricals? does music work on a Seurat object or SpatialExperiment? otherwise there is a way for the conversion without the need of transforming everything to strings. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>

Reply:
<p>No, it works with matrices. </p><p>How would you do it?</p>